tracer/src/Datadog.Trace/Util/RuntimeId.cs

```diff
@@ -0,0 +1,75 @@
+// <copyright file="RuntimeId.cs" company="Datadog">
+// Unless explicitly stated otherwise all files in this repository are licensed under the Apache 2 License.
+// This product includes software developed at Datadog (https://www.datadoghq.com/). Copyright 2017 Datadog, Inc.
+// </copyright>
+
+using System;
+using System.Runtime.InteropServices;
+using System.Threading;
+using Datadog.Trace.Logging;
+
+namespace Datadog.Trace.Util
+{
+    internal static class RuntimeId
```

Comment:
I think we should add an integration test for this (to validate that we get the same runtime id for each appdomain when running outside of IIS)

Reply:
will do

Reply:
done https://github.com/DataDog/dd-trace-dotnet/pull/2474/commits/ecb34d13642094717ad2e65e3fa0d189b80d1b06